pypesto/select/model_problem.py

```diff
@@ -165,16 +165,22 @@ def set_result(self, result: Result):
         self.model.set_criterion(Criterion.NLLH, float(self.best_start.fval))
         self.model.compute_criterion(criterion=self.criterion)
 
-        self.model.estimated_parameters = {
+        estimated_parameters = {
             id: float(value)
             for index, (id, value) in enumerate(
-                zip(
-                    self.pypesto_problem.x_names,
-                    self.best_start.x,
-                )
+                dict(
+                    zip(
+                        self.pypesto_problem.x_names,
+                        self.best_start.x,
+                    )
+                ).items()
```

Comment:
Just out of curiosity, why is using `dict` here better than just using `zip` only as before?

Reply:
Hm, not sure why that was added, it's removed now, thanks!